examples/react/src/App.js

```diff
@@ -12,9 +11,17 @@ import simpleosProvider from 'eos-transit-simpleos-provider';
 import keycatProvider from 'eos-transit-keycat-provider';
 // import portisProvider from 'eos-transit-portis-provider'
 
+// import { Scatter } from 'ual-scatter';
+// import { Lynx } from 'ual-lynx';
+// import { Ledger } from 'ual-ledger';
+// import { MeetOne } from 'ual-meetone';
+// import { TokenPocket } from 'ual-token-pocket';
+
+import LoginButton from './components/loginButton';
+
 dotenv.config();
 
-let chainNetworkForExample = 'eos_kylin';
+let chainNetworkForExample = 'eos_main';
```

Comment:
Please don't change this in the code. The sample app should point to kylin by default. Ideally, we could add a drop-down in the UX to allow the user to select the target chain visually.